### DISCUSSION TITLE
Welcome to Smart-Vehicle-Fleet-Manager Discussions!

### DISCUSSION THREAD

**Discussion by abhishekpatel946** [abhishekpatel946/Smart-Vehicle-Fleet-Manager#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Muskan-Kabirpanthi** [comment]
Hello 👋

**Reply by abhishekpatel946** [reply]
hello @Muskan-Kabirpanthi, how can i help you?

**Reply by Muskan-Kabirpanthi** [reply]
I work on a feature `Navigation` to link our different section's of our website.

**Reply by abhishekpatel946** [reply]
sure you can work on it.

**Reply by abhishekpatel946** [reply]
> I work on a feature `Navigation` to link the different sections of our website.

The issue resolved
a06718efb16f91a59da471a63bcfc3c0f80febf2

**Comment by abhishekpatel946** [comment]
Hey @Muskan-Kabirpanthi you can test our App on your windows machine.
